crmsh/constants.py

```diff
@@ -486,4 +486,11 @@
 SSH_KEY_CRMSH_TAG = "crmsh-generated"
 SSH_WITH_KEY = "ssh -i {}".format(SSH_KEY_CRMSH)
 SCP_WITH_KEY = "scp -i {}".format(SSH_KEY_CRMSH)
+KEY_NOT_EXIST_HINTS = """{key} couldn't be fetch from {peer}.
+This happens if the cluster was created with an older version of crmsh or the keys have been removed.
+Please, run the next commands on {peer}:
```

Comment:
Maybe we need to specify that these commands must be executed with admin rights (for us is obvious, but who knows). Maybe something like:
```
Please, run the next commands on {peer} with admin rights:
```

Reply:
Changed. Thanks! @arbulu89